wallets/react-wallet-v2/package.json

```diff
@@ -25,6 +27,7 @@
     "@walletconnect/sign-client": "2.9.0",
     "@walletconnect/utils": "2.9.0",
     "bs58": "5.0.0",
+    "cosmjs-types": "^0.8.0",
```

Comment:
Unrelated addition?

Reply:
I added this to resolve a typing error (should've been added to devDeps anyway). I think this originated from downgrading the `@cosmjs` packages. Removed it!